detectable/src/main/java/com/blackduck/integration/detectable/util/FileFormatChecker.java

```diff
@@ -33,7 +33,10 @@ protected FileFormatChecker(Set<String> knownFileFormats) {
     public abstract void handleUnknownVersion(@Nullable String unknownFileFormat);
 
     protected boolean checkForVersionCompatibility(Supplier<@Nullable String> getFileFormatVersion) {
-        return checkForVersionCompatibility(getFileFormatVersion.get());
+        if(getFileFormatVersion != null) {
```

Comment:
Nit: Is there an ifPresent and orElse that we could use?

Reply:
No unfortunately not available